kpi/serializers/v2/asset_counts.py

```diff
@@ -0,0 +1,23 @@
+from dateutil.relativedelta import relativedelta
+from django.utils import timezone
+from rest_framework import serializers
+
+
+class AssetCountsSerializer(serializers.Serializer):
+    daily_submission_counts = serializers.SerializerMethodField()
+    total_submission_count = serializers.SerializerMethodField()
+
+    class Meta:
+        fields = ('daily_submission_counts', 'total_submission_count')
+
+    def get_daily_submission_counts(self, asset):
+        today = timezone.now().date()
+        if 'days' in self.context:
+            start_date = today - relativedelta(days=int(self.context['days']))
+        else:
+            start_date = today - relativedelta(days=31)
+        daily_counts = asset.deployment.get_daily_counts(timeframe=(start_date, today))
+        return daily_counts
```

Comment:
We should validate that `days` is an integer and a setting instead of hard-coding 31. 

Something like

```python 
    days = self.context['days']   # it is always set is the viewset , see comment below
    try:
        days = int(days)
    except ValueError:
        raise serializers.ValidationError(t('not an integer... blah blah blah'))
    
    start_date = today - timedelta(days=days)
```